src/common/table/Table.stories.tsx

```diff
@@ -8,16 +8,16 @@ export default {
 };
 
 const MOUNTAINS = [
-  { name: 'Kilimanjaro', height: 5895, country: 'Tanzania' },
-  { name: 'Everest', height: 8848, country: 'Nepal' },
-  { name: 'Mount Fuji', height: 3776, country: 'Japan' },
-  { name: 'Mont Blanc', height: 4808, country: 'Italy/France' },
-  { name: 'Vaalserberg', height: 323, country: 'Netherlands' },
-  { name: 'Denali', height: 6168, country: 'United States' },
-  { name: 'Popocatepetl', height: 5465, country: 'Mexico' },
+  { country: 'Tanzania', height: 5895, name: 'Kilimanjaro' },
+  { country: 'Nepal', height: 8848, name: 'Everest' },
+  { country: 'Japan', height: 3776, name: 'Mount Fuji' },
+  { country: 'Italy/France', height: 4808, name: 'Mont Blanc' },
+  { country: 'Netherlands', height: 323, name: 'Vaalserberg' },
+  { country: 'United States', height: 6168, name: 'Denali' },
+  { country: 'Mexico', height: 5465, name: 'Popocatepetl' },
```

Comment:
This change has been done in https://github.com/City-of-Helsinki/berth-reservations-admin/pull/18
Please update your branch after merging https://github.com/City-of-Helsinki/berth-reservations-admin/pull/18 into develop.

Reply:
branch has been updated

Reply:
Updating this branch also included the missing peer dependencies so this PR isn't relevant anymore.